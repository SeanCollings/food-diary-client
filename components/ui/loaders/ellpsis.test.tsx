import { render } from '@testing-library/react';
import { Ellipsis } from './';

describe('<Ellipsis>', () => {
  it('should render', () => {
    const { asFragment } = render(<Ellipsis />);
    expect(asFragment()).toMatchSnapshot();
  });
});
