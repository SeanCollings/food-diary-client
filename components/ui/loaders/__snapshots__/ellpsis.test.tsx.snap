// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ellipsis> should render 1`] = `
<DocumentFragment>
  <div
    class="sc-9e136426-0 extUxe lds-ellipsis"
    height="30"
    size="12"
    spacing="8"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
